idea-plugin/src/main/kotlin/arrow/meta/ide/plugins/purity/PurityPlugin.kt

```diff
@@ -2,39 +2,73 @@ package arrow.meta.ide.plugins.purity
 
 import arrow.meta.Plugin
 import arrow.meta.ide.IdeMetaPlugin
-import arrow.meta.ide.dsl.editor.inspection.ExtendedReturnsCheck
 import arrow.meta.invoke
 import com.intellij.codeHighlighting.HighlightDisplayLevel
 import com.intellij.codeInspection.ProblemHighlightType
+import org.jetbrains.kotlin.builtins.KotlinBuiltIns
 import org.jetbrains.kotlin.codegen.coroutines.isSuspendLambdaOrLocalFunction
 import org.jetbrains.kotlin.idea.caches.resolve.resolveToDescriptorIfAny
 import org.jetbrains.kotlin.idea.util.nameIdentifierTextRangeInThis
 import org.jetbrains.kotlin.lexer.KtTokens
 import org.jetbrains.kotlin.psi.KtNamedFunction
+import org.jetbrains.kotlin.psi.KtProperty
 import org.jetbrains.kotlin.resolve.calls.tower.isSynthesized
-import org.jetbrains.kotlin.util.ReturnsCheck
+import org.jetbrains.kotlin.types.KotlinType
 
 val IdeMetaPlugin.purity: Plugin
   get() = "PurityPlugin" {
     meta(
       addApplicableInspection(
-        defaultFixText = "Suspend",
+        defaultFixText = "SuspendFunction",
```

Comment:
We need a table with all plugins and their text so we can iterate and apply it to all of them. A collaborative doc where we can decide what goes on each plugin and how we can best present the info. Pics of what it looks like now on each plugin in the column or with some Lorem ipsum placeholder text would help to asses that. This is an important part of the user experience and we have to nail it not just for this plugin but in Meta in general.

Reply:
Should we move this to an actual Doc Part in the plugins directory?
Or should the Text still stay here?
We can curate the table in an issue at first as we're doing it here #131 , but code seems more maintainable. In fact, I am adding the necessary lines to it.